### ISSUE TITLE
Trying to start with iextrading4j in Eclipse Kepler with Java 8

### ISSUE THREAD

**Issue by jamoforall** [WojciechZankowski/iextrading4j#98]
## Summary
I downloaded iextrading4j from Github and imported the code into a fresh new Eclipse Java project. 

## Details
There were so many errors that made me believe I was using the code wrongly in my project. So I have to ask, what is the workflow for importing or using iextrading4j in Eclipse?

### What you expected to happen
I want to start using the iextrading4j code without any hassle. I have tried resolving all the dependencies but they are far too many. I am also new to Maven so that is also a problem for me. I am using Java 8 SE and Eclipse Kepler

### Other Information (sample code, links, etc.)
I am proficient in core Java.

![iextrading4j](https://user-images.githubusercontent.com/3886341/91188148-422d6700-e6f1-11ea-82ba-f653e1e13244.png)

**Comment by jamoforall** [comment]
Hi,

I have also tried importing the pom.xml files using Eclipse Kepler (Java SE 8) and I only get errors as seen in the screenshot below.

![iextrading4j_pom_maven](https://user-images.githubusercontent.com/3886341/91200044-ebc72500-e6fe-11ea-8b35-46193845a590.png)

**Comment by jamoforall** [comment]
I solved the problems. The main problem was my Maven installation. I had tried to use the Maven installation provided by Eclipse Kepler's Help > Install Software.

I simply downloaded and installed Maven from https://maven.apache.org/download.cgi and then put it in my programs files folder. Then I edited the PATH to reflect the bin folder of the Maven installation. Eclipse picked it up after reinstallation and now I am able to use Maven to import and deploy the project!

Simple.
